Main.java

```diff
@@ -401,6 +394,25 @@ private static void showProfile(Client client) {
     Storage.saveClient(client);
   }
 
+  private static String getInventoryDescription(Client client) {
+      StringBuilder result = new StringBuilder("You have:\n");
+      int numValues = 0;
+      for (Map.Entry<Integer, Integer> item : client.inventory.entrySet()) {
+        numValues += item.getValue();
+        if (item.getValue() == 1) {
+          result.append(item.getValue() + " " +
```

Comment:
Maybe do all of these concats through StringBuilder?

Reply:
It's already StringBuilder